rust/kcl-lib/src/execution/mod.rs

```diff
@@ -1309,6 +1313,52 @@ impl ExecutorContext {
     }
 }
 
+#[derive(Debug, Clone, Copy, Serialize, PartialEq, Eq, Ord, PartialOrd, Hash, ts_rs::TS, JsonSchema)]
+#[ts(export_to = "Artifact.ts")]
```

Comment:
```suggestion
#[ts(export)]
```

Reply:
This was just moved, are you sure you want to remove this?

Reply:
Yes, I think I want to remove it because it determines what TS files that get generated. Before, it was sort of namespaced in with the other artifact stuff. But if you've moved it out, there shouldn't be another type with the same name exported to TS.